inventories/vagrant.py

```diff
@@ -8,9 +8,9 @@
 import sys
 
 try:
-    from StringIO import StringIO
+    from StringIO import StringIO  # pyright: reportMissingImports=false
 except ImportError:
-    from io import StringIO
+    from io import StringIO  # pyright: reportMissingImports=false
```

Comment:
I've never used pyright so I'm just wondering. This is Python 2/3 compatibility code. If we write it the other way around and first try Python 3 (from io ..), do you still need the comment?

Reply:
The order doesn't matter. Depending of the python version I use with the project one import will ever lead to a warning.
Tthe change is only an convenient thing. I guess it is too environment specific so I would remove this from PR. 

pyright is part of pylance which is used by vscode to check such things.

Reply:
My reasoning was that most environments will now be Python 3 so it makes sense to me to first try that. If that would also solve the import error, then I'd be happier with that. In general I'm not against linters or other static analysis but would like to have the minimum amount of control statements.

Reply:
> If that would also solve the import error, then I'd be happier with that.

There are no import errors, only warnings. 

without comments:

![image](https://user-images.githubusercontent.com/14936920/111362222-d3185f80-868e-11eb-98ea-e2b55824ebc3.png)

with comments:

![image](https://user-images.githubusercontent.com/14936920/111362317-ea574d00-868e-11eb-87bb-46ec8dbba7a6.png)

As I said, this is only a convenient change.